hack/jenkins/e2e.sh

```diff
@@ -612,26 +612,76 @@ case ${JOB_NAME} in
     : ${E2E_UP:="false"}
     : ${E2E_TEST:="true"}
     : ${E2E_DOWN:="false"}
-    : ${GINKGO_TEST_ARGS:="--ginkgo.focus=Skipped.*Cluster\supgrade.*upgrade-cluster"}
-    NUM_MINIONS=3
+    : ${GINKGO_TEST_ARGS:="--ginkgo.focus=Skipped.*Cluster\supgrade.*upgrade-master"}
+    : ${NUM_MINIONS:=3}
```

Comment:
Okay.  Can you file an issue (if there isn't already one) and add TODO(<issue#>)s?